string object_id
---
Result result
---
string status
